app/models/ability.rb

```diff
@@ -213,7 +213,7 @@ def initialize(user)
 
       can [:show_announcements, :organizational,
            :show_structures, :search_examples, :search, :show_random_quizzes,
-           :display_course], Lecture
+           :display_course, :subscribe_page], Lecture
```

Comment:
This does the trick only for generic users. The corresponding rules should also be added to admins/editors/teachers (their rulesets are some lines above in the other parts of the `if` block).